rules/js.txt

```diff
@@ -2270,6 +2285,16 @@
 <new FontFaceLoadStatus> = <FontFace>.status;
 <new Promise_FontFace_> = <FontFace>.load();
 <new Promise_FontFace_> = <FontFace>.loaded;
+<new sequence_FontFace_> = new Array(<FontFace>);
+
+<new FontSrc p=0.75> = "<cssproperty_font-src_val>"
+<new FontSrc> = "<cssproperty_font-src_val>", <FontSrc>
+
+<new DOMString> = <FontFace>.src;
+<FontFace>.src = <FontSrc>;
+<new FontFace> = new FontFace(<DOMString>, <FontSrc>, {});
```

Comment:
Would it make sense, instead of the <DOMString>, for the first param in constructor to be "cssproperty_font-family" instead? That way, font families in JS objects would match those in generated CSS, which I assume would help triggering some JS/CSS interaction.

Same for <FontFace>.family property above.

Reply:
Good call! Changed